surprise.py

```diff
@@ -0,0 +1,36 @@
+from functools import wraps
+from typing import Callable
+"""
+The code is a decorator, which makes the decorated function prints "Surprise!" instead of its original execution.
+"""
+
+
+def surprise_decorator(func: Callable) -> Callable:
+    """
+    The decorator returns a function which prints "Surprise!".
+    :param func: The ignored function.
+    :return: A function which prints "Surprise!".
+    """
+    @wraps(func)
+    def wrapper() -> None:
+        """
+        The Wrapper prints "Surprise!" and not execute the wrapped function.
+        :return:None.
+        """
+        print("Surprise!")
+    return wrapper
+
+
+@surprise_decorator
+def no_surprise() -> None:
+    """
+    The function prints I know it!.
+    :return: None.
+    """
+    print("I know it!")
+
+
+if __name__ == "__main__":
+    no_surprise()
+    print(no_surprise.__name__)
+    print(no_surprise.__doc__)
```

Comment:
This should not be here.
Don’t use git add -a ever, use git add -p
https://git-scm.com/docs/git-add